packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -2219,6 +2221,12 @@ class TimePickerDialog extends StatefulWidget {
   /// Callback called when the selected entry mode is changed.
   final EntryModeChangeCallback? onEntryModeChanged;
 
+  /// {@macro flutter.material.time_picker.switchToInputEntryModeIcon}
+  final Icon switchToInputEntryModeIcon;
+
+  /// {@macro flutter.material.time_picker.switchToTimerEntryModeIcon}
+  final Icon switchToTimerEntryModeIcon;
```

Comment:
Could we make these properties nullable? Revert this change to restore the defaults and just override them here

<img width="887" alt="image" src="https://github.com/user-attachments/assets/257bd564-ed2d-46d4-8068-0d4513f2d5e2">
 



This way we don't have to provide multiple parameter with defaults values 

<img width="550" alt="image" src="https://github.com/user-attachments/assets/3908b1c2-8880-48fc-b729-0459895f9804">

<img width="562" alt="image" src="https://github.com/user-attachments/assets/e89809d9-ea8f-4342-bc23-3babd68e9c3a">

<img width="582" alt="image" src="https://github.com/user-attachments/assets/50dfee68-8a16-460e-bb86-202a13005a33">